keyboards/modelh/rules.mk

```
@@ -0,0 +1,22 @@
+# MCU name
+MCU = STM32F103
+
+# GENERIC STM32F103C8T6 board - stm32duino bootloader
+OPT_DEFS = -DCORTEX_VTOR_INIT=0x2000
+MCU_LDSCRIPT = STM32F103x8_stm32duino_bootloader
+BOARD = STM32_F103_STM32DUINO
+
+DFU_ARGS = -d 1eaf:0003 -a2 -R
+DFU_SUFFIX_ARGS = -v 1eaf -p 0003
```

Pretty sure that most (if not all) of this is set by specifying the bootloader and MCU. 
```suggestion
BOOTLOADER = stm32duino
```